lint.sh

```diff
@@ -1,6 +1,24 @@
 #!/bin/bash
 
 rc=0
+venv=.lintvenv
+# exclude venv and git dir from linting
+fargs=(. -not -path \*${venv}\* -not -path \*.git\*)
+
+trap cleanup EXIT
+cleanup(){
+  deactivate
```

Comment:
The idea was to use venv for local developer testing (eg via githooks.sh), but use docker for gate jobs for speed. 

Fixed deactivate by testing for the function before calling it.